### PR TITLE
Use unpack_variants from dbus-fast

### PR DESCRIPTION
**Pull request by bdraco**
A faster `unpack_variants` is now provided by dbus-fast

### CODE REVIEW

**Comment by bdraco** [comment]
> I'm not familiar with this one, what does it mean?
> 
Subject say it all, but clearly not in this case 🤣 

> Can you please add an explanation of why we are making the change to the commit message?

Adjusted
> 
> Also, it looks like all of the dependencies were updated in poetry.lock, not just the ones related to this PR (can be fixed by reverting the changes to that file then run `poetry lock --no-update`).

Done

File: CHANGELOG.rst [hunk 1]
```diff
@@ -10,6 +10,10 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 `Unreleased`_
 =============
 
+Changed
```
Comment (dlech) [inline CHANGELOG.rst:13]: It looks like we have a duplicate changed section here.

Comment (bdraco) [reply CHANGELOG.rst]: Conflict fixed correctly this time and repushed

**Comment by dlech** [comment]
Thanks!

**Pull request merged**